plaso/analyzers/hashers/entropy.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""The entropy calculation implementation."""
+
+from __future__ import division
+from __future__ import unicode_literals
+
+import collections
+import math
+
+from plaso.analyzers.hashers import interface
+from plaso.analyzers.hashers import manager
+
+
+class EntropyHasher(interface.BaseHasher):
+  """Calculates the byte entropy of input files."""
+
+  NAME = 'entropy'
+  DESCRIPTION = 'Calculates the entropy of input data.'
+
+  def __init__(self):
+    """Initializes the entropy hasher."""
+    super(EntropyHasher, self).__init__()
+    self._counter = collections.Counter()
```

Comment:
What does this counter contain? the number of occurrences of a byte value?

Reply:
Done